src/MassTransit/DependencyInjection/Configuration/ScopedFilterSpecificationObserver.cs

```diff
@@ -36,16 +41,12 @@ public void MessageSpecificationCreated<T>(IMessageSendPipeSpecification<T> spec
             where TContext : class, PipeContext
             where T : class
         {
-            if (typeof(T).HasInterface<Fault>())
```

Comment:
I think this will be a breaking change.

Reply:
it was there for ages, right now it is done in constructor, look closer